tests/testthat/test-col_label.R

```diff
@@ -48,3 +48,17 @@ test_that("get col part from a col-label vector", {
   x <- c("a//first", "b@@second", "c//third")
   expect_identical(pmplots:::col_label_col(x), c("a", "b", "c"))
 })
+
+test_that("col_label accepts list or vector", {
+  x <- c("a//b", "c@@d")
+  a <- col_label(x)
+  b <- col_label(as.list(x))
+  expect_identical(a,b)
+})
+
+test_that("col_labels accepts list or vector", {
+  x <- c("a//b", "c@@d")
+  a <- pmplots:::col_labels(x)
+  b <- pmplots:::col_labels(as.list(x))
+  expect_identical(a,b)
+})
```

Comment:
Looks like duplicate of test on lines 52-57 (aside from `pmplots:::` qualification, which I don't think is relevant).

Reply:
Thanks, @kyleam.   

The previous test (on 52-57) was for `col_label()` (exported) and this one is for a vectorized version of that (`col_labels()`) which isn't exported.  I could probably change `col_label()` to take input with length > 1, but it is all over the code and I didn't want to risk something. 

Ok to go ahead with tests as is?

Reply:
Bah, sorry for overlooking that.  Thanks for the correction.

> Ok to go ahead with tests as is?

Yes 🤦

Reply:
Took me several minutes to figure that out too. :(